clang/lib/Driver/XRayArgs.cpp

```diff
@@ -67,8 +67,12 @@ XRayArgs::XRayArgs(const ToolChain &TC, const ArgList &Args) {
                    false)) {
     XRayShared = true;
 
-    // DSO instrumentation is currently limited to x86_64
-    if (Triple.getArch() != llvm::Triple::x86_64) {
+    // DSO instrumentation is currently limited to x86_64 and aarch64
```

Comment:
you can say "Certain targets support ..." so that this comment doesn't need to be updated again.